modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_row_with_background.yml

```
@@ -0,0 +1,286 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.az_row_with_background
+    - field.field.node.az_event.field_az_attachments
+    - field.field.node.az_event.field_az_body
+    - field.field.node.az_event.field_az_contacts
+    - field.field.node.az_event.field_az_event_category
+    - field.field.node.az_event.field_az_event_date
+    - field.field.node.az_event.field_az_featured_image_as_thumb
+    - field.field.node.az_event.field_az_link
+    - field.field.node.az_event.field_az_location
+    - field.field.node.az_event.field_az_media_thumbnail_image
+    - field.field.node.az_event.field_az_photos
+    - field.field.node.az_event.field_az_subheading
+    - field.field.node.az_event.field_az_summary
+    - field.field.node.az_event.field_az_trellis_id
+    - node.type.az_event
+  module:
+    - field_group
+    - smart_date
+    - smart_title
+    - user
+third_party_settings:
+  field_group:
+    group_link:
+      children:
+        - group_row
+      label: Link
+      parent_name: group_card
+      region: content
+      weight: 2
+      format_type: link
+      format_settings:
+        classes: text-decoration-none
+        show_empty_fields: false
+        id: ''
+        target: custom_uri
+        custom_uri: '[node:az-canonical-url]'
+        target_attribute: default
+    group_card:
+      children:
+        - group_link
+      label: Card
+      parent_name: ''
+      region: content
+      weight: 0
+      format_type: html_element
+      format_settings:
+        classes: 'card bg-gray-100 card-borderless mb-4 mt-n2 card-clickable hover'
```

Align the top of the event rows in this view display with the top of the news rows in the other view display:
```suggestion
        classes: 'card bg-gray-100 card-borderless mb-2 card-clickable hover'
```

---

I would keep `mb-4` to match news. But maybe I'm seeing the `mt-n2` needed if the views have a title? I'll have to look more into that piece.

---

Thanks @danahertzberg! We will keep the current classes since they align the rows when the view titles are present (which will be the common expected behavior). We can investigate the alignment issues further in the main PR.